lib/webpack.config.js

```diff
@@ -35,6 +46,8 @@ module.exports = {
         }
     },
     plugins: [
-
+        new webpack.ProvidePlugin({
+            'Globalize': "globalize/lib/globalize"
```

Comment:
What will happen if Globalize is not installed in node modules? Will it fail compilation?

Reply:
No, "globalize/lib/globalize" should be imported in code